### PR TITLE
remove tab info from links

### PR DESCRIPTION
**Pull request by mairaw**
Re. https://github.com/dotnet/docs/pull/16223#discussion_r357359906

/cc @Youssef1313

### CODE REVIEW

File: docs/core/get-started.md [hunk 1]
```diff
@@ -47,27 +47,27 @@ You can get started developing .NET Core applications by following these step-by
 - Watch a video on [how to install and use Visual Studio 2017 and .NET Core](https://channel9.msdn.com/Blogs/dotnet/Get-Started-NET-Core-Visual-Studio-2017/).
 - [Getting started with .NET Core using the command-line.](tutorials/cli-create-console-app.md)
 
-See the [.NET Core dependencies and requirements](install/dependencies.md?tabs=netcore30&pivots=os-windows) article for a list of the supported Windows versions.
+See the [.NET Core dependencies and requirements](./install/dependencies.md?pivots=os-windows) article for a list of the supported Windows versions.
```
Comment (tdykstra) [inline docs/core/get-started.md]: Why add "./"?

**Review by tdykstra — APPROVED** [review]
LGTM

File: docs/core/get-started.md (hunk 1 above)
Comment (mairaw) [reply docs/core/get-started.md]: For me, it's easier to see is a sub-folder with the . 

But I can undo that if the team doesn't like it.

Comment (Thraka) [reply docs/core/get-started.md]: I feel we should leave it a personal matter. Both are valid ways of the same thing.

File: docs/core/tools/using-ci-with-cli.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ The installer script is automated to run at the start of the build to fetch and
 > [!NOTE]
 > **Azure DevOps Services**
 >
-> When using the installer script, native dependencies aren't installed automatically. You must install the native dependencies if the operating system doesn't have them. For more information, see [.NET Core dependencies and requirements](../install/dependencies.md?tabs=netcore30&pivots=os-linux).
+> When using the installer script, native dependencies aren't installed automatically. You must install the native dependencies if the operating system doesn't have them. For more information, see [.NET Core dependencies and requirements](../install/dependencies.md?pivots=os-linux).
```
Comment (Thraka) [inline docs/core/tools/using-ci-with-cli.md]: There are scripts for both windows and linux. I think we should remove the specific pivot and rely on any previously chosen pivot.

```suggestion
> When using the installer script, native dependencies aren't installed automatically. You must install the native dependencies if the operating system doesn't have them. For more information, see [.NET Core dependencies and requirements](../install/dependencies.md).
```

**Review by Thraka — APPROVED** [review]
One minor thing that I think makes it better for the user in this specific article.

**Pull request merged**